static/templates/move_topic_to_stream.hbs

```diff
@@ -1,9 +1,9 @@
-<div id="move_topic_modal" class="modal modal-bg hide fade new-style" tabindex="-1" role="dialog" aria-labelledby="move_topic_modal_label" aria-hidden="true">
+<div id="move_topic_modal" class="modal move-topic-modal modal-bg hide fade new-style" tabindex="-1" role="dialog" aria-labelledby="move_topic_modal_label" aria-hidden="true">
     <div class="modal-header">
         <button type="button" class="close" data-dismiss="modal" aria-label="{{t 'Close' }}"><span aria-hidden="true">&times;</span></button>
         <h3 id="move_topic_modal_label">{{t "Move topic" }} </h3>
     </div>
-    <div class="modal-body">
+    <div class="modal-body move-topic-modal">
```

Comment:
Why do you need to add this class in two places?

Reply:
This is so because `modal-body` and `modal` classes doesn't support the overlay objects to overflow.Hence, had to explicitly mention the `modal-body` to overflow here.
Here's the GIF if an extra `overflow` CSS property isn't attached to it. 

![overflow](https://user-images.githubusercontent.com/53977614/114082549-83166c80-98cb-11eb-9222-66192c62399e.gif).